CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.24.0 Release
+- Release Date: October 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
```

Comment:
Anything we want to highlight?

Reply:
moved a couple from improvements to highlight.

Also include `Added support for copy and paste protection managed configuration for Android`